packages/material-ui-lab/package.json

```diff
@@ -52,7 +52,7 @@
     "react-is": "^16.8.0 || ^17.0.0"
   },
   "devDependencies": {
-    "@material-ui/types": "^5.1.0"
+    "@material-ui/types": "5.1.0"
```

Comment:
5.1.6 might work as well (since it contains `Omit`). But compatibility with 5.1.6 isn't tested. 5.1.0 is the version on `master` so let's stick with that.